annals-of-botany.csl

```diff
@@ -104,14 +104,25 @@
           </group>
           <text prefix=" " suffix="." macro="publisher"/>
         </if>
+        <else-if type="webpage" match="any">
+          <group suffix=". ">
+            <text macro="title" font-style="italic" prefix=" "/>
+          </group>
+          <text variable="URL" prefix=" " suffix="."/>
+          <date variable="accessed">
+            <date-part name="day" form="numeric" suffix=" "/>
+             <date-part name="month" form="short" suffix=" "/>
+             <date-part name="year" form="long"/>
+           </date>
+        </else-if>    
         <else-if type="chapter paper-conference" match="any">
           <text macro="title" prefix=" "/>
-          <group prefix=".">
+          <group prefix=" In: ">
```

Comment:
@damnation333, if possible, we'd like to use CSL terms instead of verbatim text, because terms can be automatically translated, and using them make CSL styles easier to (re)use in other languages. Here, you can change `<group prefix=" In: ">` to e.g.

```xml
<group prefix=" " >
  <text term="in" text-case="capitalize-first" suffix=": "/>
```

(Don't take my nitpicking the wrong way, by the way. We appreciate your help. I'm just pointing out some things that can be improved)